test_knowledge_base_api.py

```diff
@@ -0,0 +1,35 @@
+from knowledge_base_api import MusicKB
+
+class TestMusicKB():
+    def __init__(self):
+        self.kb_api = MusicKB()
+
+    def test_get_song_data(self):
+        self.kb_api.open_connection()
+        res = self.kb_api.get_song_data("Despacito")
+        self.kb_api.close_connection()
+
+        assert res == ('Despacito', 'Justin Bieber')
+
+    def test_get_song_data_dne(self):
+        self.kb_api.open_connection()
+        res = self.kb_api.get_song_data("Not In Database")
+        self.kb_api.close_connection()
+
+        assert res == None
+
+    def test_get_song_data_no_open_connection(self):
+        res = self.kb_api.get_song_data("Not In Database")
+        assert res == None
+
+def run_tests():
+    print("Running tests...")
+    test_obj = TestMusicKB()
+    test_obj.test_get_song_data()
+    test_obj.test_get_song_data_dne()
+    test_obj.test_get_song_data_no_open_connection()
+    print("Finished!")
+
+
+if __name__ == "__main__":
+    run_tests()
```

Comment:
Is there a script to set-up the database?  Is it possible to commit the database file so that we can test our work against a pre-populated DB?